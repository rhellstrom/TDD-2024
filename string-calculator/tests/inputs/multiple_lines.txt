scalc '1,2,3'
scalc '4,5,6'
scalc '//[***][%%%][!!!]\n1!!!2%%%%4'

