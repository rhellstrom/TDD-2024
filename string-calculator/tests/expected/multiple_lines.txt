The result is 6
The result is 15
The result is 7
